scripts/rpm_owners/packages.json

```diff
@@ -4637,7 +4637,7 @@
     },
     "xapi-clusterd": {
         "added_by": "XS",
-        "maintainer": "?",
+        "maintainer": "XAPI & Network",
         "note": "proprietary (GFS2)",
         "package": "xapi-clusterd",
         "status": "ignored"
```

Comment:
I understand for v6d and xapi-storage-plugins, for which we may want to check changes to see if our replacements need additional work (which we likely want reflected at some point in a more accurate status), but can't this one purely be ignored?